examples/regression/plot_kim2020_simulations.py

```diff
@@ -0,0 +1,457 @@
+"""
+==================================================
+Reproducing the simulations from Kim et al. (2020)
+==================================================
+
+:class:`mapie.regression.MapieRegressor` is used to reproduce the simulations
+by Kim et al. (2020) [1] in their article which introduces the
+jackknife+-after-bootstrap method.
+
+For a given model, the simulation fits MAPIE regressors with
+jackknife+ and jackknife+-after-bootstrap methods, on different resamplings of
+a data set loaded from 'https://archive.ics.uci.edu/', and compares the
+coverage levels and the width means of the PIs.
+
+In order to reproduce results from the tutorial notebook of Kim and al, we
+implemented their regression model ``Ridge2``, a variant of `sklearn.Ridge`
+with an adaptive regularization parameter.
+
+We compare jackknife+ and jackknife+-after-bootstrap, with fixed and random
```

Comment:
Maybe we can mention that the other models explored by Kim+(2020) can also be used in this script ?

Reply:
Done